docs-ref-conceptual/use-azure-cli-successfully-in-powershell.md

```diff
@@ -0,0 +1,459 @@
+
+This behavior is the same in PowerShell 5 and PowerShell 7 running in Linux or Windows.
+
+```azurecli
+# Variable block
+$randomIdentifier = (New-Guid).ToString().Substring(0,8)
+$subscriptionId="00000000-0000-0000-0000-000000000000"
+$resourceGroup="msdocs-app-rg$randomIdentifier"
+$containerRegistryName="msdocscr$randomIdentifier"
+
+# prior to this GET example, the resource group and container registry were created in the article.
+
+az rest --method get --url https://management.azure.com/subscriptions/$subscriptionId/resourceGroups/$resourceGroup/providers/Microsoft.ContainerRegistry/registries/${containerRegistryName}?api-version=2023-01-01-preview
+```
+
+---
+
+## Pass parameters containing a PowerShell special character
+
+There are special characters of PowerShell, such as at (`@`). To run Azure CLI in PowerShell, add a
+backtick `` ` `` before the special character to escape it. You can also enclose the value in single
+(`'`) or double (`"`) quotes.
+
+**The following three examples will work in PowerShell:**
+
+* parameterName `@parameters.json
+* parameterName '@parameters.json'
+* parameterName "@parameters.json"
+
+**This example will not work in PowerShell:**
+
+* parameterName @parameters.json
+
+## Pass parameters containing JSON
+
+For complex arguments like a JSON string, the best practice is to use Azure CLI's `@<file>` convention to load from a file to bypass the shell's interpretation. Note that the At ('@') symbol is a [splatting operator](/powershell/module/microsoft.powershell.core/about/about_splatting) in PowerShell, so it should be quoted.
+
+There are good examples in [az ad app create](/cli/azure/ad/app#az-ad-app-create-examples) that contain both JSON file content and command examples.  Here's a code snippet:
+
+```azurecli
+# Bash script
+az ad app create --display-name myTestAppName \
+    --is-fallback-public-client \
+    --required-resource-accesses @manifest.json
+```
+
+In this example, notice the double quotes (`"..."`) around the JSON file name needed in a PowerShell environment.
+
+```azurecli
+# PowerShell script
+az ad app create --display-name myTestAppName `
+    --is-fallback-public-client `
+    --required-resource-accesses "@manifest.json"
+```
+
+## Pass parameters containing key:value pairs
+
+Some Azure CLI parameter values, such as Azure resource tags, require key:value pairs. If your `key` or `value` contains a space or special character, the Bash and PowerShell syntax are not always the same.
+
+See [Create tags to practice quoting differences](./get-started-tutorial-2-environment-syntax.md#create-tags-to-practice-quoting-differences) in the [Learn to use the Azure CLI](./get-started-tutorial-0-before-you-begin.md) tutorial. This tutorial step gives examples for Bash, PowerShell and Cmd for the following key:value pair scenarios:
+
+* spaces
+* empty values
+* special characters
+* variables
+
+## Error handling for Azure CLI in PowerShell
```

Comment:
Is error handling something fundamental? As in, would this information be important for a beginner to know? Or is it a "nice to know" or "more info" kind of topic? The rest of the article seems set up for beginners but the error handling portion seems a bit more advanced?

Reply:
@daphnemamsft , right now, this article is intended for a catch-all of tips for customers running the Azure CLI in a PowerShell environment.